requirements.txt

```diff
@@ -1,3 +1,4 @@
 logbook==0.9.0
 pymongo==2.8
 requests==2.6.0
+pypandoc==0.9.6
```

Comment:
This dependency is already listed in requirements_dev.txt, which is where it belongs, since this is a build-time dependency, not a run-time dependency. So this change is incorrect.